time_startup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Do you intend to keep this script? If so, please move it to `scripts/` and add 3 lines of docs explaining what it is for.

Reply:
No, this was a stray.